packages/@aws-cdk/aws-codebuild/test/integ.project-vpc.ts

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+import assets = require('@aws-cdk/assets');
+import ec2 = require('@aws-cdk/aws-ec2');
+import cdk = require('@aws-cdk/cdk');
+import { Project } from '../lib';
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'aws-cdk-codebuild-project-vpc');
+const vpc = new ec2.VpcNetwork(stack, 'MyVPC');
+const securityGroup = new ec2.SecurityGroup(stack, 'SecurityGroup1', {
+    allowAllOutbound: true,
+    description: 'Example',
+    groupName: 'Bob',
+    vpc: vpc,
+});
+new Project(stack, 'MyProject', {
+    buildScriptAsset: new assets.ZipDirectoryAsset(stack, 'Bundle', { path: 'script_bundle' }),
+    securityGroups: [securityGroup],
+    vpc: vpc
```

Comment:
This needs to be just `vpc` (it's a TypeScript shorthand that our linter enforces).